### ISSUE TITLE
idea / proposal: jenkins_job -> virtual attribute to trigger a job after creation

### ISSUE THREAD

**Issue by jaceq** [taiidani/terraform-provider-jenkins#68]
Hi again @taiidani :)

So I am extending our use of your provider and starting to manage jobs via it in addition to credentials.
We have however a bit specific setup, as in we manage all our pipelines via Jenkinsfile files in respective repositories.

In order to avoid adding them to jenkins by hand, I want to be able to create a minimal job (with only repo name and jenkinsfile location) and then trigger it so it will pull jenkinsfile and re-configure itself.

Long story short I'd like to have an  attribute in 'jenkins_job" that would trigger a job run after creation - I had a quick look at gojenkins library and it seems possible to invoke a run... 
What do you think about such idea? I am happy to contribute this.

**Comment by jaceq** [comment]
Hi, thanks for responding!

So my case is rather simple I have jobs configured with a default variable so when I trigger them I don't even need to send any params. I also don't need to track their build (as the first one is a dummy build). 
So I'd like to start as simple as possible:
- add a parameter to jenkins_job
- trigger a build
- return build ID (I can keep that in state and never read / update so won't trigger any changes)

Then if there would be any further need for things like adding params etc it could be extendend.

So, are you ok with this simple, step by step approach? If so I will look into that when possible (as in when I have some time) - if on the other hand you don't like this apporach please let me know so I won't invest my time into 'not acceptable' solution :)

Thanks
